opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -2766,6 +2777,11 @@ namespace Opm
 		    case Segment::SegmentType::SICD :
 		        assembleSICDPressureEq(seg, well_state);
 			break;
+            case Segment::SegmentType::AICD : {
```

Comment:
Weird intenting? Perhaps due to use of tabs in the existing code?

Reply:
It looks like other line uses `tabs` but not this line.